src/microbe_stage/components/OrganelleContainer.cs

```diff
@@ -43,6 +43,12 @@ public struct OrganelleContainer
     [JsonIgnore]
     public List<SlimeJetComponent>? SlimeJets;
 
+    /// <summary>
+    ///   The slime jets with mucocyst
+    /// </summary>
+    [JsonIgnore]
```

Comment:
This should not be json ignore and also this needs to be set among the int variables, otherwise struct padding will waste 4 bytes of space for each struct instance.